src/vstt/task.py

```diff
@@ -282,7 +282,11 @@ def _do_target(
         mouse_pos = tm.cursor.pos
         stop_waiting_time = 0.0
         stop_target_time = 0.0
-        tm.green_target_index = None
+        tm.green_target_index = (
+            tm.green_target_index
+            if tm.green_target_index == trial["num_targets"]
+            else None
+        )
```

Comment:
this logic is a bit convoluted - it might be clearer to instead set `tm.green_target_index = None` when the next target is displayed, i.e. here:

https://github.com/ssciwr/vstt/blob/8a014de89b716e8ff2c561277bb824f05cbd1ca2/src/vstt/task.py#L346-L349

we can set `tm.green_target_index = None` and pass it to update_target_colors

Reply:
Yes looks fine - it's a bit less explicit about what's going on but has the same behaviour, since the `green_target_index` argument of update_target_colors is `None` by default